packages/fxa-settings/src/components/App/HeaderContinueTo/index.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+
+export type HeaderContinueToProps = {
+  prefixDataTestId: string;
+  prefixFtlId: string;
+  headingText: string;
+  serviceName?: string;
+};
+
+const HeaderContinueTo = ({
+  prefixDataTestId,
+  prefixFtlId,
+  headingText,
+  serviceName,
+}: HeaderContinueToProps & RouteComponentProps) => {
+  return (
+    <header data-testid={`${prefixDataTestId}-header`} className="mb-4">
+      <Localized id={`${prefixFtlId}-heading`}>
```

Comment:
> should have a class of `card-header` similar to what's in content-server now

@LZoog The original styling for the header has `overflow-hidden text-ellipsis`. Is this behaviour we want to keep? Seems that might hide important information? I find hiding overflow tends to make more sense for a paragraph when there's an option to expand to view the full content.

Reply:
@vpomerleau Are you referring to the `.card-header` style that's under `#fxa-settings-profile-header`? I don't think that one's applied here (I don't think we're using any `_settings.scss` styles anymore but ya never know in this old code), but feel free to correct me if I'm mistaken.

Reply:
Hm, I thought that was the origin of the `.card-header` styling for `reset_password`. If not there, then where? I haven't been able to find it. 🙈

Reply:
Ah, see [this comment](https://github.com/mozilla/fxa/pull/14599#discussion_r1049130067). You probably didn't find it because we're nesting it with `&-header`. (Pros and cons of nesting.)

Reply:
@LZoog I've updated the styling - it now uses fxa-react's existing classes. There are some differences in styling vs the original content-server view (especially font-weight). I think it looks good as is, but let me know if you'd like me to tweak the styling to more closely match the original view (you can compare how it looks in the screenshots from this PR's description).